LICENSE

```diff
@@ -200,3 +200,16 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+CppLint is:
```

Comment:
Marcin: Thank you for making the changes. One comment:
Please replace this line with "BSD-3 License".  We don't want anything specific associated the license in the LICENSE file.

Reply:
done :)